cloud/deploy/deploy.go

```diff
@@ -473,7 +473,11 @@ func buildImageWithoutDags(path string, imageHandler airflow.ImageHandler) error
 
 		dagsIgnoreSet = true
 	}
-
+	// remove astro-run-dag from requirments.txt
```

Comment:
I don't think it is a good idea to be doing this in `deploy.go`, always better to handle it where it is being used/created makes the code easier to understand and less complex

Reply:
+1

Reply:
okay I fixed it